.travis.yml

```
@@ -119,22 +126,18 @@ jobs:
       <<: *STANDARD_TEST_JOB
       php: nightly
       env: DRIVER="phpdbg"
-
       script:
-        # We don't have xdebug on the nightly build, so we always run with phpdbg.
-        - make compile;
-        - ./tests/e2e_tests build/infection.phar;
+        # We don't have Xdebug on the nightly build, so we always run with PHPDBG.
```

```suggestion
        # We don't have Xdebug on the nightly build, so we always run with phpdbg.
```

this is how it's named in https://github.com/krakjoe/phpdbg#installation